files/image_config/pcie-mon/pcie-mon.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Moniter the PCIe device presence and status
```

Comment:
Typo: "Monitor". Also, this service is no longer a periodic monitor, but rather a one-time check, correct?